pkg/doc/bbs/bbs12381g2pub/internal/field_element.go

```diff
@@ -0,0 +1,343 @@
+/*
+Taken from https://github.com/kilic/bls12-381
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
```

Comment:
Please add comments on what commit this file is from and what changes have been made to this file.
Please add link to the LICENSE file from the original repo (at the same commit level of this file).